client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -134,6 +136,66 @@ export const useCommandsArrayWpcom = ( {
 		displayNotice( __( 'SSH/SFTP password reset and copied to clipboard.' ) );
 	};
 
+	const getEdgeCacheStatus = async ( siteId: number ) => {
+		const response = await wpcom.req.get( {
+			path: `/sites/${ siteId }/hosting/edge-cache/active`,
+			apiNamespace: 'wpcom/v2',
+		} );
+
+		return response;
+	};
+
+	const clearEdgeCache = async ( siteId: number ) => {
+		try {
+			const response = await getEdgeCacheStatus( siteId );
+
+			if ( response ) {
+				// If global cache is active, purge the cache
+				await wpcom.req.post( {
+					path: `/sites/${ siteId }/hosting/edge-cache/purge`,
+					apiNamespace: 'wpcom/v2',
+				} );
+				displayNotice( __( 'Succesfully cleared cache.' ) );
```

Comment:
```suggestion
				displayNotice( __( 'Cache cleared.' ) );
```

Reply:
@danielbachhuber - FIY, the `clearWordPressCache` has a notice that says `Successfully cleared WordPress cache`. I am happy to change this text but was wondering if we should keep it consistent between the two ways to clear cache

Reply:
While I don't love that message, I'm fine to keep the message consistent between the two.